src/server/service/tracing.rs

```diff
@@ -0,0 +1,338 @@
+    pub kv_pessimistic_rollback: ReadableDuration,
+    pub kv_commit: ReadableDuration,
+    pub kv_cleanup: ReadableDuration,
+    pub kv_batch_get: ReadableDuration,
+    pub kv_batch_rollback: ReadableDuration,
+    pub kv_txn_heart_beat: ReadableDuration,
+    pub kv_check_txn_status: ReadableDuration,
+    pub kv_check_secondary_locks: ReadableDuration,
+    pub kv_scan_lock: ReadableDuration,
+    pub kv_resolve_lock: ReadableDuration,
+    pub kv_delete_range: ReadableDuration,
+    pub kv_import: ReadableDuration,
+    pub kv_gc: ReadableDuration,
+    pub coprocessor: ReadableDuration,
+    pub mvcc_get_by_key: ReadableDuration,
+    pub mvcc_get_by_start_ts: ReadableDuration,
+    pub raw_get: ReadableDuration,
+    pub raw_batch_get: ReadableDuration,
+    pub raw_scan: ReadableDuration,
+    pub raw_batch_scan: ReadableDuration,
+    pub raw_put: ReadableDuration,
+    pub raw_batch_put: ReadableDuration,
+    pub raw_delete: ReadableDuration,
+    pub raw_batch_delete: ReadableDuration,
+    pub raw_delete_range: ReadableDuration,
+    pub raw_get_key_ttl: ReadableDuration,
+    pub raw_compare_and_swap: ReadableDuration,
+    pub raw_checksum: ReadableDuration,
+    pub raw_coprocessor: ReadableDuration,
+    pub invalid: ReadableDuration,
+}
+
+impl Default for TracingConfig {
+    fn default() -> Self {
+        TracingConfig {
+            enable: false,
+            max_span_count: 100,
+            duration_threshold: TracingDurationThreshold::default(),
+        }
+    }
+}
+
+impl Default for TracingDurationThreshold {
+    fn default() -> Self {
+        TracingDurationThreshold {
+            kv_get: ReadableDuration(Duration::from_millis(300)),
+            kv_scan: ReadableDuration(Duration::from_millis(300)),
+            kv_prewrite: ReadableDuration(Duration::from_millis(300)),
+            kv_pessimistic_lock: ReadableDuration(Duration::from_millis(300)),
+            kv_pessimistic_rollback: ReadableDuration(Duration::from_millis(300)),
+            kv_commit: ReadableDuration(Duration::from_millis(300)),
+            kv_cleanup: ReadableDuration(Duration::from_millis(300)),
+            kv_batch_get: ReadableDuration(Duration::from_millis(300)),
+            kv_batch_rollback: ReadableDuration(Duration::from_millis(300)),
+            kv_txn_heart_beat: ReadableDuration(Duration::from_millis(300)),
+            kv_check_txn_status: ReadableDuration(Duration::from_millis(300)),
+            kv_check_secondary_locks: ReadableDuration(Duration::from_millis(300)),
+            kv_scan_lock: ReadableDuration(Duration::from_millis(300)),
+            kv_resolve_lock: ReadableDuration(Duration::from_millis(300)),
+            kv_delete_range: ReadableDuration(Duration::from_millis(300)),
+            kv_import: ReadableDuration(Duration::from_millis(300)),
+            kv_gc: ReadableDuration(Duration::from_millis(300)),
+            coprocessor: ReadableDuration(Duration::from_secs(1)),
```

Comment:
Recommended to be 300ms, as 300ms is already considered as a slow query in TiDB.